gulpfile.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Maybe you can change this file from 'xxx.js' to 'xxx.ts'?

Reply:
That could be done using the TS node I'll give it a try.